modules/ibi-create-standalone-config-iso.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="create-standalone-config-iso_{context}"]
-= Deploying a {sno} cluster using the {product-title} installation program
+= Deploying a {sno} cluster using the `openshift-install` program
```

Comment:
```suggestion
= Deploying a {sno} cluster using the openshift-install program
```

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings